appserver/extras/docker-images/server-full/src/main/docker/Dockerfile

```diff
@@ -35,26 +37,26 @@ ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini \
 RUN true \
     # Install dependencies
     && apt-get update \
-    && apt-get install -y coreutils gpg \
+    && apt-get install -y gpg \
     && rm -rf /var/lib/apt/lists/* \
     # Create and set the Payara user and working directory owned by the new user
-    && groupadd --gid 1000 payara \
-    && useradd -u 1000 -M -s /bin/bash -d ${HOME_DIR} payara -g payara \
+    && addgroup --gid 1000 payara \
+    && adduser --system --uid 1000 --no-create-home --shell /bin/bash --home "${HOME_DIR}" --gecos "" --ingroup payara payara \
     && echo payara:payara | chpasswd \
     && mkdir -p ${DEPLOY_DIR} \
     && mkdir -p ${CONFIG_DIR} \
     && mkdir -p ${SCRIPT_DIR} \
     && chown -R payara: ${HOME_DIR} \
     # Verify tini
```

Comment:
If you guys don't want gpg et al in the resulting image, this could as well be done in a multistage build step.

Reply:
Caches are disabled in this build, but this leads me to a question - is it possible to tell docker to use cache only for some stage?
Enabled cache lead few versions back to the state where docker image of 5.194 contained 5.193 inside it.

Reply:
Now it should be better - I created separate module for the shared basic image. Only the micro has it's own, because it is based on alpine.